db/dbformat.h

```diff
@@ -549,6 +551,10 @@ class InternalKeySliceTransform : public SliceTransform {
     return transform_->InRange(user_key);
   }
 
+  virtual size_t FullLength() const override {
+    return transform_->FullLength();
+  }
```

Comment:
Shouldn't make this unnecessary virtual function call. Directly call the user prefix extractor where it is needed.